public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -584,21 +582,21 @@ table(width="100%")
       code-example.
         &lt;tr ng-repeat="movie in movieList | orderBy : 'title'">
       :marked
-        Orders the collection, as specified by the expression.
+        `OrderBy` orders the collection, as specified by the expression.
         In this example, the movie title orders the movieList.
     td
       :marked
         ### none
         For performance reasons, no comparable pipe exists in Angular&nbsp;2.
-        Order/sort results in the component.
-        If the same ordering/sorting code will be reused in several templates, consider building a custom pipe.
+        Instead, use component code to order or sort results. If you need the same ordering or sorting code in several templates, consider building a custom pipe.
```

Comment:
As above, I wonder whether it's really necessary to use both "order" and "sort" here. Feel free to research further with the writer... or to leave this as is.